packages/webapp/src/components/Task/AddSoilAmendmentProducts/ProductCard/index.tsx

```diff
@@ -182,7 +185,13 @@ const SoilAmendmentProductCard = ({
           />
         </>
       )}
-      {/* TODO: LF-4249 <QuantityApplicationRate /> */}
+      <QuantityApplicationRate
+        productId={PRODUCT_ID}
```

Comment:
I absolutely did! I was only using this to pass to the `<Collapse />` id, and since it was functioning with the fixed string, I think passing product isn't necessary at all. @Duncan-Brain in the other PR I will remove the `productId` prop.